### PR TITLE
Create traditional chinese (tw) docs channel

### LINKED ISSUE #7348: REQUEST: New Slack <[channel|usergroup|bot|token|webhook]> <[channel|usergroup|bot|token|webhook] name>
### GitHub Username

@kaiyee0

### Slack Username

Kay Lin

### What Type of Request is it (Channel, User Group, Bot, Token, or Webhook)?

Channel

### Name of Requested Resource

kubernetes-docs-tw

### Description of Request

Create a Slack channel for Traditional Chinese (zh-tw) docs localiztion.

/sig docs
/language zh-tw

### PR DESCRIPTION
**Pull request by kaiyee0**
**Which issue(s) this PR fixes**:

Fixes #7348

### CODE REVIEW

File: communication/slack-config/sig-docs/docs-channels.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ channels:
   - name: kubernetes-docs-pt
   - name: kubernetes-docs-ru
   - name: kubernetes-docs-ta
+  - name: kubernetes-docs-tw
```
Comment (sftim) [inline communication/slack-config/sig-docs/docs-channels.yaml:19]: The language code for (traditional) Chinese is `zh`. We shouldn't make this change as-is.

Comment (kaiyee0) [reply communication/slack-config/sig-docs/docs-channels.yaml:19]: Hi @sftim , I understand that you're suggesting not to make the change as the language code "zh" yet it already exists for **Simplified Chinese**. Do you get suggested replacement name for the channel name?

Comment (sftim) [reply communication/slack-config/sig-docs/docs-channels.yaml:19]: We have two options:

- use the existing channel
- rename the existing channel to `kubernetes-docs-zh-cn` and make a new channel `kubernetes-docs-zh-tw`

Even with the geopolitics involved, I would use the existing channel name and not make a new one. If that doesn't work out, we can consider a split.

Comment (kaiyee0) [reply communication/slack-config/sig-docs/docs-channels.yaml:19]: Got it. In that case, I will try to use the existing channel first. Thank you for your kind reply.

**Comment by kaiyee0** [comment]
As there's already a zh code channel, we decided to first use the existing channel.

**Pull request closed without merging**